Scenarios/AKS-Secure-Baseline-PrivateCluster/Apps/SimpleGMSAApp/manifests/deployment_sampleapp.yml

```
@@ -85,7 +86,7 @@ spec:
               - key: agentpool
                 operator: In
                 values:
-                - wnp
+                - < Windows Node Pool >
```

Any reason to not just set it to "winpl" -- which is what it's currently called?

```suggestion
                - winpl
```